invesalius/navigation/image.py

```diff
@@ -20,55 +20,90 @@
 import numpy as np
 
 import invesalius.session as ses
+import invesalius.project as prj
+import invesalius.constants as const
 from invesalius.pubsub import pub as Publisher
 
 
-class Image():
+class Image:
     def __init__(self):
-        self.image_fiducials = np.full([3, 3], np.nan)
-
+        self.__bind_events()
         self.LoadState()
 
-    def SaveState(self):
-        state = {
-            'image_fiducials': self.image_fiducials.tolist(),
-        }
-        session = ses.Session()
-        session.SetState('image', state)
+    @property
+    def fiducials(self):
+        return prj.Project().image_fiducials
 
-    def LoadState(self):
-        session = ses.Session()
-        state = session.GetState('image')
+    def __bind_events(self):
+        Publisher.subscribe(self.OnStateProject, 'Enable state project')
+        pass
 
-        if state is None:
-            return
+    def SaveState(self):
+        pass
+        # state = {
```

Comment:
No need to change it for this patch to go in, but just for future: leaving commented-out code in the codebase is often considered a major anti-pattern. In short, if someone needs access to this code later on, it can be found in Git history. For a longer explanation, see, e.g. https://methodpoet.com/commented-out-code/

But there is currently commented-out code in many places in InVesalius codebase; I think cleaning it up can be done when modifying the related parts of the code; hence, this particular change is not really changing the current situation in any major way.

Reply:
Agreed!